experimenter/experimenter/nimbus_ui_new/templates/nimbus_experiments/detail.html

```diff
@@ -1,14 +1,341 @@
+            <tr>
+              <th>Feature config</th>
+              <td>
+                {% for feature in experiment.feature_configs.all %}
+                  {{ feature.name }}- {{ feature.description }}
+                {% empty %}
+                  <span style="color: red;">Not set</span>
+                {% endfor %}
+              </td>
+              <th>Advanced targeting</th>
+              <td>{{ experiment.targeting_config.name }}- {{ experiment.targeting_config.description }}</td>
+            </tr>
+            <tr>
+              <th>Hypothesis</th>
+              <td colspan="3">{{ experiment.hypothesis }}</td>
+            </tr>
+            <tr>
+              <th>Primary Outcomes</th>
+              <td colspan="3">
+                {% for outcome, url in primary_outcome_links %}
+                  <a href="{{ url }}" target="_blank" rel="noopener noreferrer">{{ outcome|format_to_title }}</a>
+                  {% if not forloop.last %},{% endif %}
+                {% empty %}
+                  <span style="color: red; ">Not set</span>
+                {% endfor %}
+              </td>
+            </tr>
+            <tr>
+              <th>Secondary Outcomes</th>
+              <td colspan="3">
+                {% for outcome, url in secondary_outcome_links %}
+                  <a href="{{ url }}" target="_blank" rel="noopener noreferrer">{{ outcome|format_to_title }}</a>
+                  {% if not forloop.last %},{% endif %}
+                {% empty %}
+                  <span style="color: red; ">Not set</span>
+                {% endfor %}
+              </td>
+            </tr>
+            <tr>
+              <th>Team projects</th>
+              <td>
+                {{ experiment.projects.all|join:"<br>"|format_not_set|safe }}
+              </td>
+              <th>Subscribers</th>
+              <td>
+                {{ experiment.subscribers.all|join:"<br>"|format_not_set|safe }}
+              </td>
+            </tr>
+          </tbody>
+        </table>
+      </div>
+    </div>
+    <!-- Risk Mitigation Questions Card -->
+    <div class="card mb-3">
+      <div class="card-header">
+        <h4>Risk Mitigation Questions</h4>
+      </div>
+      <div class="card-body">
+        <table class="table table-striped">
+          <tbody>
+            <tr>
+              <th colspan=3>{{ RISK_QUESTIONS.BRAND }}</th>
+              <td style="{% if experiment.risk_brand|yesno == 'Yes' %}color: red;
+                         font-weight: bold;
+                         {% endif %}">{{ experiment.risk_brand|yesno|format_not_set|safe }}</td>
+            </tr>
+            <tr>
+              <th colspan="3">{{ RISK_QUESTIONS.MESSAGE }} Message Consult</th>
+              <td style="{% if experiment.risk_message|yesno == 'Yes' %}color: red;
+                         font-weight: bold;
+                         {% endif %}">{{ experiment.risk_message|yesno|format_not_set|safe }}</td>
+            </tr>
```

Comment:
We should be able to make this simpler, like [this](https://github.com/mozilla/experimenter/blob/main/experimenter/experimenter/legacy/legacy-ui/templates/experiments/risk_inline.html#L1-L6)